pkgs/build-support/rust/build-rust-crate/default.nix

```diff
@@ -172,6 +221,7 @@ stdenv.mkDerivation (rec {
 
     src = crate.src or (fetchCrate { inherit (crate) crateName version sha256; });
     name = "rust_${crate.crateName}-${crate.version}${lib.optionalString buildTests_ "-test"}";
+    version = crate.version or null;
```

Comment:
Is the `or null` case actually required? Do we have crates without a version? Looking one line above we are already using the version as part of the `name` attribute. I would assume it is not optional.

Reply:
I guess not. I wanted to be defensive and did not see that I could already reasonably expect the version to be required.